idl/token.idl.hh

```diff
@@ -6,7 +6,7 @@ class token {
         after_all_keys,
     };
     dht::token::kind _kind;
-    bytes _data;
+    bytes data();
 };
```

Comment:
const. I suggest renaming it to make it clear it's a non-native operation now, e.g. `to_bytes()`.

Maybe returning a bytes_view is sufficient.

Reply:
Scratch this comment, it is in the idl, and the code is correct.

Reply:
However, we should rename token::data() to discourage its use, can be done later.